src/sendError.js

```diff
@@ -0,0 +1,11 @@
+const sendError = (response, statusCode, errorMessages) => {
+  const errors = [];
+
+  errorMessages.forEach(error => {
```

Comment:
Just push your errors in `paramsCheck` like `{ message: text }` instead do extra iteration here